### DISCUSSION TITLE
Sort list by commit size

### DISCUSSION THREAD

**Discussion by stevenvachon** [community/community#17275] [Pull Requests]
One might be searching for a contrastively large or small pull request and sorting by such narrows it down.

Please sort by the number of lines changed

**Comment by mtschoen-unity** [comment]
+1 to this! I'd like to also be able to _see_ the number of changes in the PR list as well. I'm often looking for small PRs to review/merge when I have spare cycles. It would be nice to know at-a-glance how many files/lines were changed by a particular PR from the PR list view.

**Comment by Avasam** [comment]
Related: https://github.com/orgs/community/discussions/78897 and https://github.com/orgs/community/discussions/21726
